payments/index.md

```diff
@@ -1,8 +1,209 @@
+2. You also need to create a `<script>` source within the container, using the href value you obtained when submitting your initial POST. As mentioned above, all Hosted View operations have a rel description beginning with  `"view-"`. The example below is taken from the view-payment operation, above - enabling Swish payments through Hosted View.
+
+```HTML
+<script id="paymentPageScript" src="https://ecom.stage.payex.com/swish/core/scripts/client/px.swish.client.js?
+                                    token=bcff0db777d5bcf21a210235342921f46da993efa5e91340f713c8cedf4aac38"></script>
+```
+3. Lastly, you must initiate the Hosted View with a JavaScript call to open the Hosted View iframe embedded on you site.
+
+```HTML
+JavaScript Call
+<script language="javascript">
+    payex.hostedView.page(configuration).open();
+</script>
+```
+See the technical overview of each payment method and the technical reference for more specific information.
+
+### Redirect implementation
+The Redirect implementation lets you redirect your customer to an easy-to-use PCI compliant payment platform, hosted by PayEx and available from both web and mobile browsers. The consumer selects a payment method and proceeds to hosted payment pages.
+
+```
+TODO: Insert image showing the flow of redirect implementation
+```
+
+Please visit our [demoshop](https://ecom.externalintegration.payex.com/pspdemoshop) to view our Payment Menu and Redirect implementation in action. Use the configuration below:
+
+```
+TODO: Insert pickture from demoshop showing the following configuration:  
+PayEx Payment Pages  
+Redirect  
+Norway  
+English
+```
+
+#### Getting started
+To start using the Redirect platform, you need to implement the appropriate operation returned from the initial API Request, containing the URL that the customer's browser needs to be redirected to.
+
+The combination of properties should be similar to all payment methods. In the example below, the href attribute refers to the redirect URL and the rel description redirect-sale indicate that the redirect scenario will generate a one-phased sales transaction. A two-phase credit card payment would during the same scenario generate an authorization transaction, and the  rel description would in that case be `"redirect-authorization"`.
+
+```JS
+{
+   "operations": [
+        {
+           "method": "GET",
+           "href": "https://ecom.externalintegration.payex.com/swish/payments/sales
+                     /993b479653da83671c074316c7455da05fced9d634431edbb64f3c5f80a863f0",
+           "rel": "redirect-sale"
+        }
+    ]
+}
+```
+See the technical overview of each payment method and the technical reference for more specific information.
+
+## <a name="the-payment-process">The Payment Process</a>
+
+
+The core payment process of the eCommerce APIs are the same across all payment instruments.
+
+1. ### Creating a payment
```

Comment:
Headers in lists are a weird construction. Let's just use headers for the top-level items and keep the lists underneath each header.

```suggestion
### Creating a payment
```